app/models/reports/all_subscribers.rb

```diff
@@ -3,17 +3,17 @@ class AllSubscribers < Report
   def initialize(output_options = {})
     @view_params = {
       :name => 'All subscribers',
-      :current_season => Time.this_season
+      :vouchertypes => Vouchertype.subscription_vouchertypes()
     }
     super
   end
 
   def generate(params = {})
-    seasons = params[:seasons]
+    vouchertypes = Report.list_of_ints_from_multiselect(params[:vouchertypes])
     @relation = 
-      if seasons.empty?
+      if vouchertypes.empty? 
       then Customer.purchased_any_vouchertypes(Vouchertype.subscription_vouchertypes.map(&:id))
```

Comment:
this doesn't make sense.  if no vouchertypes were specified, the report runs anyway?? 
shouldn't the result here be `Customer.none`?

Reply:
The intention was to preserve the behavior when no seasons were selected, which defaults to a select all, so we left line 15 unchanged. `Customer.none` is indeed more intuitive. Should we make this change?

Reply:
ok, i see.  it's probably fine to leave the behavior as-is then.  thanks for clarifying.